autogen/agentchat/conversable_agent.py

```diff
@@ -43,9 +43,7 @@ class ConversableAgent(Agent):
     To customize the initial message when a conversation starts, override `generate_init_message` method.
     """
 
-    DEFAULT_CONFIG = {
-        "model": DEFAULT_MODEL,
-    }
+    DEFAULT_CONFIG = {}  # An empty configuration
```

Comment:
I'll be very careful with this change because we want to offer a good agent out of the box. With #1082, this change is not necessary.

Reply:
I would strongly recommend handling these defaults by providing a partially filled-in OAI_CONFIG_LIST example file, rather than building it into the code like this.

Reply:
Also, the error would persist if people accidentally over-filter with filter_dict -- I'm debugging exactly that error now in the test_utils.py oia test.

Reply:
Makes sense. What about adding a comment in the example file too about the guideline of using weaker models, such as the necessity of modifying the system message and safety risk?

Reply:
@sonichi Great point! I added a note both to the OAI_CONFIG_LIST_sample, and the README.md